content/en/developers/ide_plugins/vscode/_index.md

```diff
@@ -77,7 +77,19 @@ Install the extension either directly in the IDE, or from the web:
 
 - **From the web**: Install from the extension's page on [Visual Studio Marketplace][1].
 
+### MCP Server setup
+
+<div class="alert alert-info">The Datadog MCP Server is in Preview. Complete <a href="https://www.datadoghq.com/product-preview/datadog-mcp-server">this form</a> to request access.</div>
+
+The extension includes access to the [Datadog Model Context Protocol (MCP) Server][3]. Ensure the MCP Server is enabled to enhance the editor's AI capabilities with your specific Datadog environment:
+
+1. Open the chat panel, select Agent mode, and click the **Configure Tools** button.
```

Comment:
```suggestion
1. Open the chat panel, select agent mode, and click the **Configure Tools** button.
```
This doesn't refer to the Datadog Agent, so I think it's okay to match the UI in having it be lowercase?

Reply:
It's not visible in the PR diff, but the screenshot does show Agent capitalized (it's the VS Code UI, not Datadog, so it's referring to Agent vs. Ask mode for the AI chat):

<img width="400" height="352" alt="image" src="https://github.com/user-attachments/assets/ea4d649d-94f0-408a-ae5f-460263133414" />

Reply:
Oh that's funny, I was looking above that at `in agent mode`! Haha happy not to die on that hill either way

Reply:
Sounds good, thanks for the detective work! Happy to fling around as many approvals as you need